2-mandatory/2-retrying-random-numbers.js

```diff
@@ -11,8 +11,16 @@ function generateRandomNumber() {
 
 function getRandomNumberGreaterThan50() {
     // TODO - implement using a do-while loop
+    let result;
+    do {
+        result = generateRandomNumber();
+    } while (result <= 50); 
```

Comment:
This is just a matter of opinion, but woulnt it be more effiecient just writing:
```suggestion
while(result <= 50)
{
        result = generateRandomNumber();
}
```